assets/src/edit-story/components/canvas/gridview/index.js

```diff
@@ -201,7 +201,7 @@ function ThumbnailSizeControl({ value, onChange }) {
         disabled={value === max}
         aria-label={__('Increase thumbnail size', 'web-stories')}
       >
-        <RectangleIcon />
+        <Rectangle />
```

Comment:
To fix the issue @miina found:

```suggestion
        <RectangleIcon />
```